src/components/ServiceRequestForm.jsx

```diff
@@ -292,7 +292,7 @@ const ServiceRequestForm = (props, errors, touched) => {
 				localProps.setFieldValue('streetAddress', addressParts[0]);
 				localProps.setFieldValue('city', addressParts[1]);
 				localProps.setFieldValue('zipCode', addressParts[3]);
-				if(localProps.requiresLocation === true){
+				if(localProps.values.requiresLocation === true){
```

Comment:
if this is a bool, this should be

```js
if (localProps.values.requiresLocation) {
...
```

Reply:
fixed this along with all others throughout app